androidresource-checker-qual/build.gradle

```diff
@@ -0,0 +1,43 @@
+plugins {
+    id 'java'
+}
+
+repositories {
+    jcenter()
+}
+
+dependencies {
+    implementation 'org.checkerframework:checker:2.9.0'
+}
+
+task copySources(type: Copy) {
+    description 'Copy androidresource-checker-qual source from the project'
+    includeEmptyDirs = false
+    doFirst {
+        // Delete the directory in case a previously copied file should no longer be in checker-qual
+        delete file('src/main/java')
+    }
+
+    from files('../src/main/java')
+    include '**/org/checkerframework/**/qual/**'
+
+    // Make files read only.
+    fileMode(0444)
+
+    into file('src/main/java')
+}
+
+jar {
+    dependsOn copySources
+    manifest {
+        attributes('Export-Package': '*')
+    }
+}
+
+compileJava {
+    dependsOn copySources
+}
+
+clean {
+    delete file('generated-src/')

```

Comment:
Above on line 27 it uses `into file('src/main/java')`. Shouldn't those two be consistent?
Can you test whether the `clean` target actually works correctly?
If this needs to be fixed, can you also make a PR against the Checker Framework sources.

Reply:
I have replaced the `generated-src` to `src`. I will make a PR to the CF sources.

Reply:
I have added the corresponding PR to the main CF repo, [link](https://github.com/typetools/checker-framework/pull/2688).